tests/tests_common.h

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file tests/tests_common.h
+ * @brief Common declarations.
+ */
+#pragma once
+#include <gtest/gtest.h>
+
+#include <src/globals.h>
+#include <src/logging.h>
+#include <src/platform/common.h>
+
+template <class T>
+struct PlatformTestSuite: testing::Test {
+  static std::unique_ptr<platf::deinit_t> &
+  get_platform_deinit() {
```

Comment:
Is there a reason why this is not just a static member?

Reply:
Honestly? This is one is always guaranteed to work. Trying to mess with `inline static` variables in curiously recurring template pattern class? Probably will too, but I had my doubts.

Reply:
Actually. this got me thinking. Will open a follow-up PR that will slightly simplify and improve things.

Reply:
Oh, it's not merged. Can do it here then.